src/webgpu/api/validation/texture/bgra8unorm_storage.spec.ts

```diff
@@ -101,6 +101,12 @@ Test that it is invalid to configure a GPUCanvasContext to 'GPUStorageBinding' u
         }
         return usageSet;
       })
+      .unless(({ usage }) => {
+        return (
+          (usage & GPUConst.TextureUsage.TRANSIENT_ATTACHMENT) !== 0 &&
+          !IsValidTransientAttachmentUsage(usage)
```

Comment:
This ends up being used in the same way in most places. Should the helper just be `isValidTextureUsageCombination()`?